.docs/README.md

```diff
@@ -312,50 +288,43 @@ $payment = $service->restorePayment([
 ]);
 ```
 
-Na objektu platby lze zavolat dvě kontrolní metody: `isFraud()` a `isPaid()`.
-První nás informuje, jestli je platba pravá, respektive nejedná-li se
-o podvrh (interně se zde kontroluje ona čtveřice parametrů předaných z platební
-brány).
+Two control methods can be called on a payment object: `isFraud ()` and `isPaid ()`. The first informs us whether the payment is genuine or not for fraud (internally, the four parameters passed from the payment system are checked by payment gate).
```

Comment:
Two control methods can be called on a payment object: `isFraud()` and `isPaid()`. The first informs us whether the payment is genuine or not for fraud (internally, the four parameters passed from the payment system are checked by payment gate).